### PR TITLE
get translation to work on data-on-text & data-off-text for boolean f…

### PR DESCRIPTION
**Pull request by aldf**
…ield type

### CODE REVIEW

File: resources/views/switch.twig [hunk 1]
```diff
@@ -7,8 +7,8 @@
             type="checkbox"
             name="{{ field_type.input_name }}"
             {{ field_type.value ? 'checked' }}
-            data-on-text="{{ trans(field_type.config.on_text ?: 'anomaly.field_type.boolean::choice.yes') }}"
-            data-off-text="{{ trans(field_type.config.off_text ?: 'anomaly.field_type.boolean::choice.no') }}"
+            data-on-text="{{ trans(field_type.config.on_text ? 'anomaly.field_type.boolean::choice.yes' : field_type.config.on_text) }}"
+            data-off-text="{{ trans(field_type.config.off_text ? 'anomaly.field_type.boolean::choice.no' : field_type.config.on_text) }}"
```
Comment (RyanThompson) [inline resources/views/switch.twig:11]: These are backwards. This should use the config text if available - otherwise fallback to the trans string.

**Review by RyanThompson — CHANGES_REQUESTED** [review]
Little bit of a logic issue here - the trans ternary is backwards.

File: resources/views/switch.twig (hunk 1 above)
Comment (aldf) [reply resources/views/switch.twig:11]: Correct ,but that won't display switch button translation for on and off ,i tried to put them in backwards, it only display YES and NO , no matter the local language you choose

**Comment by RyanThompson** [comment]
This had the value in the config array on the field type: https://github.com/anomalylabs/boolean-field_type/commit/86b720489c1802d9b64d94583ebd107c4b3614af

**Pull request closed without merging**